src/cts/src/HTreeBuilder.cpp

```diff
@@ -367,7 +375,7 @@ void HTreeBuilder::computeLevelTopology(unsigned level,
     remainingLength = previousLevel.getRemainingLength();
   }
 
-  const unsigned SLEW_THRESHOLD = options_->getMaxSlew();
+  const unsigned SLEW_THRESHOLD = techChar_->getMaxSlew(); // The normalized value of the slew of the characterization buffer (max slew)
```

Comment:
Why is this stored in two different places?  Can we consolidate this to just one?

Reply:
Yeah makes since to make a single source for that, actually, PostCtsOpt was still using the max from options_ too.